script/Ciel/Day 10/10_01_CIEL10_3_2.txt

```diff
@@ -16,92 +16,102 @@ I quickly call out to Senpai.
 }
 [sha:ae49049a648806f778c34bd95bec5c7fa39f8afe]{
 -- Page 162, Offset 27506.
 -- 先輩は今の言葉だけで俺の考えを察してくれたらしい。
-It looks like Senpai picked up on where I was going just from those few words.
+It looks like I said enough for Senpai to understand what it is I want to talk about.
 }
 [sha:727a5e6326248a58cf28cbd0005e74327fcf8fb6]{
 -- Page 162, Offset 27507.
 -- 「仕方ありませんね。５時限目をサボタージュする事になりますが、単位は大丈夫ですか？」
-"So be it then. You'll be skipping fifth grade, that won't affect your credits, I hope?"
+"So be it then.
+ You'll be skipping fifth period――I trust that won't jeopardize your attendance record?"
 }
-[sha:d4d8692e77a00aa7c28143a1ae2767ca14227176]{
+[offset:27508]{
 -- Page 163, Offset 27508.
 -- もちろん、と頷く。
-I nod in agreement.
+I shake my head. Not an issue.
 }
 [sha:660d84360ce492f2a670109092bce4237b0a1696]{
 -- Page 163, Offset 27509.
 -- 先輩は俺とは無関係を装って、部活棟に向かった。
-Senpai walks toward the wing where the club rooms are, while pretending not to know me.
+Senpai casually heads for the wing where the club rooms are located,
+ acting like the two of us don't know each other.
 }
 [sha:a9f665a50188cd63562479952b12cf3685022c3d]{
 -- Page 163, Offset 27510.
 -- ……なるほど。学校で怪しまれないように活動するには、こういう気配りが必要なんだな。
-I see... It's important indeed to pay attention to things like this if she wants to operate inside the school without raising suspicions.
+I see...
+ I guess that's the sort of attention to detail you need to avoid calling attention to yourself while undercover at school.
 }
 [sha:31722b876eef214ce30df9f8e66263ede113e46d]{
 -- Page 164, Offset 27511.
 -- ５時限目の開始を告げるチャイムが鳴っている。
-The chime announcing the beginning of fifth period rings out.
+The bell rings, announcing the beginning of fifth period.
 }
 [sha:63091d09ebd36e98b26d41145e33407997a8421a]{
 -- Page 164, Offset 27512.
 -- こんな堂々と授業をさぼるのは初めてだ。なにしろ同じ学校内である。
-It's the first time that I've so boldly skipped class, and what's more, skipped class while remaining inside the school.
+I've never been this bold about skipping class before.
+ Not only that, but I'm skipping them while still %{i}at%{/i} school.
 }
 [sha:259a61e3164bfff9c5390427c8f97769e621771b]{
 -- Page 165, Offset 27513.
 -- 「ご心配なく。先生方は茶道室に近づきませんから、見つかる事はありませんよ」
-"No need to worry, Tohno-kun. You won't be caught since the teachers don't come near the tea ceremony room."
+"No need to worry about being caught.
+ Teachers never pass by the tea ceremony room."
 }
 [sha:dd33192aeb287f72107320969881e2986e6d1d4c]{
 -- Page 166, Offset 27514.
 -- シエル先輩は台所でお茶の準備をしている。
-Ciel-senpai is preparing tea inside the kitchenette.
+Ciel-senpai is in the kitchenette brewing some tea.
 }
 [sha:4582e4c21ee9e200802460bd67008b6b9227e942]{
 -- Page 166, Offset 27515.
 -- ……あの落ち着きようからして、授業を抜け出すのは一度目や二度目ではないと見た。
-Judging from how composed she seems, it's clear that it's not the first or second time she's slipped away from class.
+She seems pretty calm about this...
+ I'd wager this isn't the first――or even the second――time she's slipped away from class.
```

Comment:
――or even the second―― doesn't really fit in when you're using a set expression like this. "not the first time" tends to imply you've done it a lot of times.

>I'd wager she has slipped away from class more than once or twice.